Feature/#24 Create user page

StepanovRoma opened the pull request:
1. Slightly changed design
2. Path constants have been added
3. The settings button and the path to it were also added
![image](https://user-images.githubusercontent.com/60768837/171560151-48fa9d20-025a-448b-b204-0a9bdd6f8ff9.png)

StepanovRoma commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Create user page](https://trello.com/c/oczeGg4i/24-create-user-page)

MangriMen commented on src/consts/index.ts:
```diff
@@ -0,0 +1 @@
+export * from './paths'
```
Need newline at end of file

MangriMen commented on src/components/user/UserCardComponent.tsx:
```diff
@@ -75,6 +78,9 @@ function UserCardComponent() {
           </Typography>
         </Box>
+        <Box>
+          <Button onClick={() => navigate(ROUTE.SETTINGS)}>Settings</Button>
```
It would be better here to use Link (ParLink as styled version of course) component instead of button, because its actually link. And you won't need a navigate.

MangriMen commented on src/components/user/UserCardComponent.tsx:
```diff
@@ -54,7 +56,7 @@ function UserCardComponent() {
           }}
           children={stringAvatar(String(user?.displayName))}
-        ></Avatar>
+        />
```
I think need pull from develop and replace Avatar component by ParAvatar.

MangriMen commented on src/components/user/UserCardComponent.tsx:
```diff
@@ -66,6 +68,7 @@ function UserCardComponent() {
               xs: '28px',
             },
             flexWrap: 'wrap',
+            padding: '2vh',
```
Padding units in vh? Why not rem?

StepanovRoma replied:
I don't know why 🥺

MangriMen commented on src/components/user/UserCardComponent.tsx:
```diff
@@ -27,16 +29,16 @@ function stringAvatar(name: string) {
     <>
       <Box
         sx={{
           display: 'flex',
```
Maybe something like this? I'm not sure, but it looks better and more space for a nickname.
![image](https://user-images.githubusercontent.com/49300253/171587274-999dbdb9-70c0-458c-bba3-738a50675f02.png)

MangriMen reviewed (changes requested):
Overall good but needs a few changes.

FFreestyler reviewed (commented):
![image](https://user-images.githubusercontent.com/49301790/171598802-a4082ce9-8b41-42af-b5bc-ac559676d6e0.png)
Maybe move settings button under AvatarBox, and wrap this text in ParButton component? >Todo: make ParButton component

StepanovRoma commented:
> ![image](https://user-images.githubusercontent.com/49301790/171598802-a4082ce9-8b41-42af-b5bc-ac559676d6e0.png) Maybe move settings button under AvatarBox, and wrap this text in ParButton component? >Todo: make ParButton component

I don't think so, because there will be a lot of empty right

FFreestyler commented:
![image](https://user-images.githubusercontent.com/49301790/171607158-91eebde1-df88-4648-8b38-9cb77618fad9.png)
But it doesn't look cool

StepanovRoma commented:
> ![image](https://user-images.githubusercontent.com/49301790/171607158-91eebde1-df88-4648-8b38-9cb77618fad9.png) But it doesn't look cool

Since this is a modal thing, then this is not a problem, in the future we can make a dimming, but it seems to be normal anyway

StepanovRoma commented:
Now it looks like this
![image](https://user-images.githubusercontent.com/60768837/171613116-342fc641-1924-4379-801d-0615bf54473b.png)

FFreestyler commented:
Why "K" in avatar so small?

StepanovRoma commented:
Most likely this is a bug related to the use of ParLink and it is not directly related to the user profile

MangriMen commented:
> Most likely this is a bug related to the use of ParLink and it is not directly related to the user profile

ParAvatar*, font size should depend on container size.

StepanovRoma commented:
1. Moved settings button to the bottom
2. Added field with other information
3. Added character description
4. Added cards with user characters
5. Fixed bag with ParAvatar fontSize
![image](https://user-images.githubusercontent.com/60768837/171803588-eca34802-2c92-4239-9cd1-a8b75d88a878.png)
